### PR TITLE
Updated pricing page and related pages

### LINKED ISSUE #4789: [COPY UPDATE] plans-and-pricing - "Ubuntu plans and pricing" - UA4I
Please update plans-and-pricing - "Ubuntu plans and pricing" from the [copy doc](https://docs.google.com/document/d/1FS1slJOfWXLmPob1eXrQ32DU4s6WP4oyGoT3t5E4W8Y/edit?usp=drivesdk)

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Updated all text and tables on /support/plans-and-pricing
- Added variables for ticks to not have to keep typing the tick image
- Replaced all *.html pages tick image tags

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/pricing](http://0.0.0.0:8001/support/plans-and-pricing)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Compare to the [copy doc](https://docs.google.com/document/d/1FS1slJOfWXLmPob1eXrQ32DU4s6WP4oyGoT3t5E4W8Y/edit#)

## Issue / Card

Fixes #4789

### CODE REVIEW

File: templates/shared/pricing/_ua-for-infrastructure.html [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+        </tr>
+
+      </tbody>
+    </table>
+  </div>
+</div>
+
+<div class="row" id="fn-ua-i-storage">
+  <div class="col-12">
+    <p id="fn-ua-i-storage"><small><sup>*</sup> For current allowance limits, see our <a href="/legal/ubuntu-advantage-service-description">service description</a>.</small></p>
```
Comment (anthonydillon) [inline templates/shared/pricing/_ua-for-infrastructure.html]: This id is duplicated two lines above

File: templates/support/plans-and-pricing.html [hunk 2]
```diff
@@ -10,41 +10,43 @@
...
+        <li class="p-list__item is-ticked">Extended Security Maintenance (ESM) provides critical security updates after an LTS release transitions from the LTS Basic Security Maintenance</li>
+        <li class="p-list__item is-ticked">Landscape system management simplifies use of Ubuntu at scale</li>
+        <li class="p-list__item is-ticked">Livepatch enables automatic kernel security hotfixes without rebooting</li>
+        <li class="p-list__item is-ticked">Knowledge Base offers a private archive of expert-written articles and tutorials</li>
+        <li class="p-list__item is-ticked">Phone and web-based support at multiple service levels</li>
+        <li class="p-list__item is-ticked">FIPS certified modules to meet Common Criteria compliance</li>
+        <li class="p-list__item is-ticked">Ceph and Swift storage, backup and recovery capabilities</li>
       </ul>
-      <p><a href="http://buy.ubuntu.com" class="p-button"><span class="p-link--external">Visit the Ubuntu Advantage store</span></a></p>
+      {% comment %}<p><a href="http://buy.ubuntu.com" class="p-button"><span class="p-link--external">Visit the Ubuntu Advantage store</span></a></p>{% endcomment %}
```
Comment (anthonydillon) [inline templates/support/plans-and-pricing.html]: Can we not remove this mark up instead of commenting it?

Comment (pmahnke) [reply templates/support/plans-and-pricing.html]: we are meant to add it back soon, but if it bothers you I can

File: templates/support/plans-and-pricing.html [hunk 3]
```diff
@@ -53,19 +55,29 @@ <h2>Ubuntu Advantage for Infrastructure</h2>
...
+      <h3 id="ua-i-virtual">Managed service options</h3>
+      <p>With our managed service options, Canonical experts will operate OpenStack clouds and Kubernetes clusters, managing your physical and virtual infrastructure.</p>
+    </div>
+  </div>
+
+  {% include "shared/pricing/_ua-for-infrastructure-options.html" %}
+
+  <div class="row">
+    <div class="col-12">
+      <p><a href="/support/contact-us?product=support-pricing" class="p-button js-invoke-modal">Contact us&nbsp;&rsaquo;</a></p>
```
Comment (anthonydillon) [inline templates/support/plans-and-pricing.html]: Button with >. Should be a link or remove the >

File: templates/support/plans-and-pricing.html [hunk 4]
```diff
@@ -76,30 +88,46 @@ <h3 id="ua-i-virtual">Ubuntu Advantage for Infrastructure - Virtual</h3>
...
-      <h2 id="desktop">For desktops</h2>
+      <h2 id="desktop">Ubuntu Advantage for desktops</h2>
     </div>
   </div>
+
   {% include "shared/pricing/_ua-desktop-support.html" %}
+
+  <div class="row">
+    <div class="col-12">
+      <p><a href="/support/contact-us?product=support-pricing" class="p-button js-invoke-modal">Contact us&nbsp;&rsaquo;</a></p>
```
Comment (anthonydillon) [inline templates/support/plans-and-pricing.html]: Button with chevron

File: templates/shared/pricing/_maas-support.html [hunk 5]
```diff
@@ -17,15 +17,15 @@
...
-          <td class="p-table__cell--highlight u-align--center">$5 per month</td>
-          <td class="p-table__cell--highlight u-align--center">$10 per month</td>
+          <td class="p-table__cell--highlight u-align--center">$5</td>
+          <td class="p-table__cell--highlight u-align--center">$10</td>
         </tr>
 
         <tr>
           <td>
-            Per region per year<br>
+            Per-region per-year<br>
```
Comment (anthonydillon) [inline templates/shared/pricing/_maas-support.html]: Price per-region, per-year
(unlimited machines*)

File: templates/support/plans-and-pricing.html [hunk 6]
```diff
@@ -138,36 +167,76 @@ <h3>AI/ML Add-on for Discoverer and Discoverer Plus</h3>
...
   </div>
 </section>
 
-<section class="p-strip--light is-deep">
+<section id="landscape" class="p-strip--light is-bordered is-deep">
   <div class="row">
     <div class="col-12">
-      <h2>Landscape Enterprise Systems Management (on-premises)</h2>
-      <p>Reduce your team&rsquo;s efforts on day-to-day management with Landscape, the most cost-effective tool to support and monitor large and growing networks of desktops, servers and clouds.</p>
+      <h2>Landscape: enterprise systems management on-premises</h2>
```
Comment (anthonydillon) [inline templates/support/plans-and-pricing.html]: "on premises"

**Comment by anthonydillon** [comment]
Sorry, QA found a few issues.

**Comment by anthonydillon** [comment]
LGTM :+1:

**Pull request merged**